streams/src/main/java/org/apache/kafka/streams/KeyQueryMetadata.java

```diff
@@ -50,32 +50,65 @@ public KeyQueryMetadata(final HostInfo activeHost, final Set<HostInfo> standbyHo
     }
 
     /**
-     * Get the Active streams instance for given key
+     * Get the active Kafka Streams instance for given key.
      *
      * @return active instance's {@link HostInfo}
+     * @deprecated Use {@link #activeHost()} instead.
      */
+    @Deprecated
     public HostInfo getActiveHost() {
         return activeHost;
     }
 
     /**
-     * Get the Streams instances that host the key as standbys
+     * Get the Kafka Streams instances that host the key as standbys.
      *
      * @return set of standby {@link HostInfo} or a empty set, if no standbys are configured
+     * @deprecated Use {@link #standbyHosts()} instead.
      */
+    @Deprecated
     public Set<HostInfo> getStandbyHosts() {
         return standbyHosts;
     }
 
     /**
-     * Get the Store partition corresponding to the key.
+     * Get the store partition corresponding to the key.
      *
      * @return store partition number
+     * @deprecated Use {@link #partition()} instead.
      */
+    @Deprecated
     public int getPartition() {
         return partition;
     }
 
+    /**
+     * Get the active Kafka Streams instance for given key.
+     *
+     * @return active instance's {@link HostInfo}
+     */
+    public HostInfo activeHost() {
+        return activeHost;
+    }
+
+    /**
+     * Get the Kafka Streams instances that host the key as standbys.
+     *
+     * @return set of standby {@link HostInfo} or a empty set, if no standbys are configured
+     */
+    public Set<HostInfo> standbyHosts() {
+        return standbyHosts;
+    }
+
+    /**
+     * Get the store partition corresponding to the key.
+     *
+     * @return store partition number
```

Comment:
I feel it should be "key partition number" instead of store.

Reply:
We are fetching the state store partition corresponding to the key with this getter and hence that description I believe.
More over, this is a copy paste work, we copied existing getter's with a new name and deprecated the old getter's. 
Any specific reason why you feel that the description should be "key partition number" ?

Reply:
@mjsax Please correct me if I'm wrong.

Reply:
I wrote the old getters & javadocs but now I feel that key partition number better conveys the meaning than store partition number.
The class KeyQueryMetadata contains the information for a key being requested via query. So, the partition number we are returning here is the partition to which the key belongs, it has no relation to any store yet. You will of course use this partition information to fetch the data corresponding to the key from any store but still that doesn't change the fact that this partition number is related to the key rather than the store.
I am fine either way though.

Reply:
IMHO, "key partition number" sound like as if the key would be partitioned, and we get the partition number for the "key partition". But in fact, the store is partitioned and thus I think the current "store partition number" is ok as-is.

Reply:
Sure, that's fine.